ui-lib/src/main/java/co/electriccoin/zcash/ui/common/repository/FlexaRepository.kt

```diff
@@ -94,27 +96,30 @@ class FlexaRepositoryImpl(
             )
         }
 
-    private fun createFlexaAccount(zecBalance: Double) =
-        AssetAccount(
-            displayName = "",
-            icon = "https://flexa.network/static/4bbb1733b3ef41240ca0f0675502c4f7/d8419/flexa-logo%403x.png",
-            availableAssets =
-                listOf(
-                    AvailableAsset(
-                        assetId = "bip122:00040fe8ec8471911baa1db1266ea15d/slip44:133",
-                        balance = zecBalance,
-                        symbol = "ZEC",
-                        icon = "https://cryptologos.cc/logos/zcash-zec-logo.png"
-                    )
-                ),
-            custodyModel = CustodyModel.LOCAL,
-            assetAccountHash = UUID.randomUUID().toString().toSha256()
-        )
+    private fun createFlexaAccount(
+        total: Double,
+        available: Double
+    ) = AssetAccount(
+        displayName = "",
+        icon = "https://flexa.network/static/4bbb1733b3ef41240ca0f0675502c4f7/d8419/flexa-logo%403x.png",
+        availableAssets =
+            listOf(
+                AvailableAsset(
+                    assetId = "bip122:00040fe8ec8471911baa1db1266ea15d/slip44:133",
+                    balance = total,
+                    balanceAvailable = available,
+                    symbol = "ZEC",
+                    icon = "https://z.cash/wp-content/uploads/2023/11/Brandmark-Yellow.png",
```

Comment:
Is it necessary to provide a remote resource link here? Could we use a more future-proof local resource?

Reply:
we cannot unfortunately :(